gramlib/plexing.mli

```diff
@@ -32,7 +32,7 @@ module type S = sig
     val get : unit -> t
     val drop : unit -> unit
     val get_comments : t -> ((int * int) * string) list
-    val set_loc_offset : int -> unit
+    val set_loc_offset : (int * int * int) -> unit
```

Comment:
Please use named parameters instead of a tuple.

Reply:
I tried to do that a month ago but wasn't able to get it compile.  IIRC it was a combination of visibility of the type and circular linking.

Reply:
I am just suggesting to do `set_loc_offset : bl:int -> el:int -> ....`

Reply:
Indeed, I misinterpreted your comment, thinking about records instead.